### PR TITLE
EVAL0051 - Option to load in annotated video rather than raw

### PR DESCRIPTION
**Pull request by dyfanmo**
## Tasks 
Option to load in annotated video rather than raw

## Symptoms 
You can not load the annotated video into the labelling.

## Problems 
The labelling tool only loads the full video.

## Solution 
I added a checkbox to select which type of video to load.

## Verification 
Script works

### CODE REVIEW

File: pavs.py [hunk 1]
```diff
@@ -378,15 +383,22 @@ def UiComponents(self, classes_label_path):
...
 
             if self.video_file_path == "":
                 try:
-                    filename = get_video_filename_from_api(self.userId, self.videoResultId)
+                    if b1_is_checked:
+                        filename = "annotated_video.mp4"
+                    elif b2_is_checked:
+                        filename = get_video_filename_from_api(self.userId, self.videoResultId)
+                    else:
+                        showErrorDialog("Choose which type of video to load.")
```
[the remark this answers is not in the document]
Comment (dyfanmo) [reply pavs.py:400]: Okay, so by default we should load the annotated video, is that correct?

[the remark this answers is not in the document]
Comment (vlad-ursu123) [reply pavs.py]: Yes

File: pavs.py [hunk 2]
```diff
@@ -126,6 +127,8 @@ def __init__(self, parent=None):
 
         self.userId = QLineEdit(self)
         self.videoResultId = QLineEdit(self)
+        self.annotated_video_radio_button = QRadioButton("Annotated Video", self)
```
Comment (vlad-ursu123) [inline pavs.py:130]: After creating it, you want to have it checked as well, so there's a visual indicator of the logic in the if statement down below. You can set it using `setChecked()` on the button object.

Comment (dyfanmo) [reply pavs.py:130]: In that case, I don't need to use "else" in the if statement below. I could use "elif" annotated_video_radio_button.is_checked().  Do you think using that will work better?

Comment (vlad-ursu123) [reply pavs.py:130]: That's fine.

**Review by vlad-ursu123 — APPROVED** [review]
Looks good now

**Pull request merged**